guides/preprocessing_layers.py

```diff
@@ -74,10 +76,9 @@
 a sample of the training data. The list of stateful preprocessing layers is:
 
 - `TextVectorization`: holds a mapping between string tokens and integer indices
-- `Normalization`: holds the mean and standard deviation of the features
-- `StringLookup` and `IntegerLookup`: hold a mapping between input values and output
+- `StringLookup` and `IntegerLookup`: hold a mapping between input values and integer
 indices.
-- `CategoryEncoding`: holds an index of input values.
+- `Normalization`: holds the mean and standard deviation of the features
```

Comment:
Nit: add a full stop

```suggestion
- `Normalization`: holds the mean and standard deviation of the features.
```